build.gradle

```diff
@@ -317,6 +317,11 @@ task sparkJar(type: ShadowJar) {
     description = "Create a combined jar of project and runtime dependencies that excludes provided spark dependencies"
     configurations = [project.configurations.sparkConfiguration]
     classifier = 'spark'
+
+    //Those have to be excluded to prevent "Invalid signature file digest for Manifest main attributes"
+    exclude 'META-INF/*.SF'
+    exclude 'META-INF/*.DSA'
+    exclude 'META-INF/*.RSA'
```

Comment:
Can you explain why this is an issue only for the spark jar (and not, say, the shadow jar)?